packages/sdk/src/types/VariableTypes.ts

```diff
@@ -43,6 +43,7 @@ export interface Variable {
     isReadonly: boolean;
     isRequired: boolean;
     occurrences: number;
+    placeholder?: string;
```

Comment:
Not really a comment to this PR but I know we use optional as the default for nullables in the engine in a lot of those models.

The engine models that are not explictly set as `includeIfNull` will shorten the model json by removing those nullable properties.
In such situations, my understanding is that reaching `placeholder` will return `undefined`; but what about a scenario where the engine is explictly serializing `null`, I assume the model will break since we are in strict mode!?

Anyone to enlighten me?

Reply:
It has been working fine so far but I feel like we should slowly make the union type the default (we have a lot of cases) if only to be more explicit and aligned with engine models (it's probably flutter team's job though).